src/http/plugins/db.ts

```diff
@@ -29,7 +29,8 @@ export const db = fastifyPlugin(async (fastify) => {
 
   fastify.addHook('preHandler', async (request) => {
     const adminUser = await getServiceKeyUser(request.tenantId)
-    const userPayload = await verifyJWT<{ role?: string }>(request.jwt, adminUser.jwtSecret)
+    const userPayload =
+      request.jwtPayload ?? (await verifyJWT<{ role?: string }>(request.jwt, adminUser.jwtSecret))
```

Comment:
if you notice, the `userPayload` is either `request.jwtPayload` which is always set only when the payload was already verified from the `jwt` plugin. 

In case we use the `db` plugin but we forgot to add the `jwt` plugin `userPayload` will be `undefined` and the verification will run